O/openfhe_julia/common.jl

```diff
@@ -0,0 +1,107 @@
+using BinaryBuilder, Pkg
+
+include("../../L/libjulia/common.jl")
+
+function prepare_openfhe_julia_build(name::String, git_hash::String)
+    # See https://github.com/JuliaLang/Pkg.jl/issues/2942
+    # Once this Pkg issue is resolved, this must be removed
+    uuid = Base.UUID("a83860b7-747b-57cf-bf1f-3e79990d037f")
+    delete!(Pkg.Types.get_last_stdlibs(v"1.6.3"), uuid)
+
+    # Collection of sources required to complete build
+    sources = [
+        GitSource("https://github.com/hpsc-lab/openfhe-julia.git",
+                  git_hash),
+    ]
+
+    # Bash recipe for building across all platforms
+    script = raw"""
+    cd $WORKSPACE/srcdir/openfhe-julia/
+
+    if [[ "${target}" == *-mingw* ]]; then
+        # This is needed because otherwise we get unusable binaries (error "The specified
+        # executable is not a valid application for this OS platform"). These come from
+        # CompilerSupportLibraries_jll.
+        # xref: https://github.com/JuliaPackaging/Yggdrasil/issues/7904
+        #
+        # The remove path pattern matches `lib/gcc/<triple>/<major>/`, where `<triple>` is the
+        # platform triplet and `<major>` is the GCC major version with which CSL was built
+        # xref: https://github.com/JuliaPackaging/Yggdrasil/pull/7535
+        #
+        # However, before CSL v1.1, these files were located in just `lib/`, thus we clean this
+        # directory as well.
+        if test -n "$(find $prefix/lib/gcc/*mingw*/*/libgcc*)"; then
+            rm $prefix/lib/gcc/*mingw*/*/libgcc* $prefix/lib/gcc/*mingw*/*/libmsvcrt*
+        elif test -n "$(find $prefix/lib/libgcc*)"; then
+            rm $prefix/lib/libgcc* $prefix/lib/libmsvcrt*
+        else
+            echo "Could not find any libraries to remove :-/"
+            find $prefix/lib
+        fi
+    fi
+    
+    # For MacOS additional linker flag is required to link typeid(__int128)
+    if [[ "$target" == *-apple-darwin* ]]; then
+        export LDFLAGS="-lc++abi"
+    fi
+    export LDFLAGS="-lc++abi"
+    # For MacOS additional linker flag is required to link typeid(__int128)
+    if [[ "$target" != *-apple-darwin* ]]; then
+        export LDFLAGS=""
+    fi
```

Comment:
@ArseniyKholod this seems wrong?

Reply:
Now it is actually fails only on x86_64, but aarch64 is OK

Reply:
And I tested only aarch86 in our repo